.github/workflows/main.yml

```
@@ -59,11 +59,128 @@ jobs:
       run: |
         pytest -v
 
+    - name: Upload coverage to Codecov
+      if: matrix.python == '3.7' && github.repository == 'allenai/allennlp-server' && (github.event_name == 'push' || github.event_name == 'pull_request')
+      uses: codecov/codecov-action@v1
+      with:
+        file: ./coverage.xml
+        # Ignore codecov failures as the codecov server is not
+        # very reliable but we don't want to report a failure
+        # in the github UI just because the coverage report failed to
+        # be published.
+        fail_ci_if_error: false
+
     - name: Clean up
       if: always()
       run: |
         pip uninstall -y allennlp-server
 
+  build_package:
+    name: Build package
+    if: github.repository == 'allenai/allennlp-server'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
```

Let's be consistent with 3.8:

```suggestion
          python-version: 3.8
```